src/v2/connectors/repository/helpers.js

```diff
@@ -18,12 +17,17 @@ exports.findOne = async (bookshelfModel, idKey, id, withRelated = []) =>
 exports.findAll = async (bookshelfModel, idKey, id) => {
   const result = await bookshelfModel
     .forge()
-    .where({ [snakeCase(idKey)]: id })
+    .where({ [camelToSnakeCase(idKey)]: id })
     .fetchAll({ require: false })
 
   return result.toJSON()
 }
 
+function camelToSnakeCase (key) {
+  const result = key.replace(/([A-Z])/g, ' $1')
```

Comment:
Again, would be great to have a brief comment that explains how this regex is working.